src/Components/Main/index.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from "react";
+import data from "../../data";
+class Main extends Component {
+  state = {
+    score: 0,
+    timer: 60,
+    counter: 0,
+    currentImg: data[0].img,
```

Comment:
I think you can cut the currentImg and name as you can get this from the "answer"? If you can convince me otherwise?